docs/en/docs/configurations/template.md

```diff
@@ -3,7 +3,7 @@
 TemplateConfig is a simple set of configurations that when passed enables the template engine.
 
 !!! info
-    Currently Esmerald supports `Jinja2` and `Mako`.
+    Currently Esmerald supports `Jinja2` or `mako` (deprecated).
```

Comment:
Currently Esmerald supports `Jinja2` or `mako` (deprecated). Mako will be removed in the release 3.6.0 of Esmerald.